stdlib/SharedArrays/test/runtests.jl

```diff
@@ -1,5 +1,5 @@
 using Test, SharedArrays
-include("../../../../test/testenv.jl")
+include(joinpath(JULIA_HOME,"..","share/julia/test/testenv.jl"))
```

Comment:
spaces after commas, and each path segment should be a separate argument (because Windows likes its `\`)